tools/buildgen/plugins/verify_duplicate_sources.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 gRPC authors.
```

Comment:
Nit:
should be 
"Copyright 2020 The gRPC Authors"

Reply:
I missed this comment but not sure what sentence is a correct one. We have a couple of variations.

Reply:
I'll update this later with other scripts.